css/style.css

```diff
@@ -0,0 +1,112 @@
+main {
+  margin: 30px auto;
+  width: 1024px;
+  box-shadow: 0px 0px 20px 1px rgba(0, 0, 0, 0.5);
+  padding: 10px 35px;
+  position: relative;
+}
+.main-info {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid black;
+  margin-bottom: 10px;
+  padding-bottom: 20px;
+}
+.main-info__title {
+  display: flex;
+  flex-direction: column;
+}
+.contacts__list{
+  list-style: none;
+}
+.contacts__item {
+  display: flex;
+  justify-content: flex-end
+}
+
+.link:hover {
+  text-decoration: none
+}
+.title{
+  text-transform: capitalize;
+  margin-bottom: 20px;
+}
+
+.photo {
+  border-radius: 10%;
+}
+
+.summary {
+  width: 300px
+}
+
+
+/* technology-list */
+.technology-list{
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  padding-left: 0px;
+}
+.technology-list__item {
+  color: #721c24;
+  background-color: #f8d7da;
+  border-color: #f5c6cb;
+  padding: 10px 20px;
+  margin-right: 10px;
+  margin-bottom: 10px;
+  text-transform: uppercase;
+  box-shadow: 4px 5px 19px 0px rgba(156, 140, 140, 0.5);
+}
+/* cv-item */
+
+.cv-item{
+  display: flex;
+  flex-direction: column;
+}
+.cv-item__title{
+  font-size: 22px;
+}
+
+.cv-item__title::before{
+  display: block;
+  position: absolute;
+  background: #721c24;
+  width: 15px;
+  height: 30px;
+  content: '';
+  left: 0px;
+}
+
+.cv-item__subtitle {
+  font-size: 18px;
+}
+
+.cv-item__additional{
+  display: flex;
+  color: gray;
+}
+
+.cv-item__subtitle {
+  text-transform: capitalize;
+}
+
+.last-child {
+  margin-left: auto;
+}
+
+.language__list {
+  list-style: none;
+  padding-left: 0px;
+}
+
+.visually-hidden {
+	position: absolute !important;
```

Comment:
Bad idea to override majority of styles using !important. If you really need such styles it's better don't use h2 and then re-style it, you can use simple div or span, etc. It looks better then 5 "!important".